application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/notification/di/NotificationConfiguration.kt

```diff
@@ -27,6 +28,9 @@ import org.springframework.context.annotation.Configuration
 )
 class NotificationConfiguration {
 
+    @Bean("notification-webhook-database-request")
+    fun notificationWebhookDatabaseRequest(): DatabaseRequest = DatabaseRequest("notification-webhook")
+
```

Comment:
This is actually a DB required by the "reporting" module, not the "notification" API.